<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSpellCheckingPage</class>
 <widget class="QWidget" name="EditorSpellCheckingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>666</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure editor spell checking options&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string>&lt;font color=&quot;#FF0000&quot;&gt;Spell checking with PyEnchant is not available.&lt;/font&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="spellingFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkingEnabledCheckBox">
        <property name="toolTip">
         <string>Select to enable spell checking</string>
        </property>
        <property name="text">
         <string>Spell checking enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Defaults</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Default language:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="defaultLanguageCombo">
           <property name="toolTip">
            <string>Select the default language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>353</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Spell checking options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="stringsOnlyCheckBox">
           <property name="toolTip">
            <string>Select to check strings only</string>
           </property>
           <property name="text">
            <string>Spell check strings only</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Minimum word size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="minimumWordSizeSlider">
             <property name="toolTip">
              <string>Move to set the minimum size of words to be checked</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lCDNumber">
             <property name="toolTip">
              <string>Displays the minimum size of words to be checked</string>
             </property>
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="intValue" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Colours</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabel2_2_2_2_2_2">
           <property name="text">
            <string>Marker Colour:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="spellingMarkerButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select the colour for the spelling markers.</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>348</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Personal lists</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Personal word list file:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="pwlEdit">
           <property name="toolTip">
            <string>Enter the filename of the personal word list</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Personal exclude list file:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="pelEdit">
           <property name="toolTip">
            <string>Enter the filename of the personal exclude list</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&lt;b&gt;Note:&lt;/b&gt; leave these entries empty to use the default</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;b&gt;Note:&lt;/b&gt; valid for all newly opened editors</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="pwlButton">
           <property name="toolTip">
            <string>Select the personal word list file via a file selection dialog</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="pelButton">
           <property name="toolTip">
            <string>Select the personal exclude list file via a file selection dialog</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Automatic spell checking</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="enabledCheckBox">
           <property name="toolTip">
            <string>Select to enable spellchecking</string>
           </property>
           <property name="text">
            <string>Automatic spell checking enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Amount of lines to autocheck at once:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="chunkSizeSpinBox">
             <property name="toolTip">
              <string>Enter the number of lines to check per go. Higher values increase checking speed but decrease GUI responsivenes</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>558</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkingEnabledCheckBox</tabstop>
  <tabstop>defaultLanguageCombo</tabstop>
  <tabstop>stringsOnlyCheckBox</tabstop>
  <tabstop>minimumWordSizeSlider</tabstop>
  <tabstop>spellingMarkerButton</tabstop>
  <tabstop>pwlEdit</tabstop>
  <tabstop>pwlButton</tabstop>
  <tabstop>pelEdit</tabstop>
  <tabstop>pelButton</tabstop>
  <tabstop>enabledCheckBox</tabstop>
  <tabstop>chunkSizeSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>minimumWordSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lCDNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
